boards/b-l475e-iot01a/board.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_b-l475e-iot01a
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the b-l475e-iot01a board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "periph/gpio.h"
+
+void board_init(void)
+{
+    /* initialize the CPU */
+    cpu_init();
+
+    /* initialize the boards LED */
+#ifdef AUTO_INIT_LED0
```

Comment:
just to make sure: is this also the case on this board or is this just a copy/paste error from the nucleo code?

Reply:
It's also the case on this board => LED_0 connected to SPI_0 clock

Reply:
> is this just a copy/paste error from the nucleo code?

hopefully all my code is not only copy paste